### ISSUE TITLE
Windows: crash on start when no Bluetooth adapter exists

### ISSUE THREAD

**Issue by flo80** [Navideck/universal_ble#18]
In windows, when using the plugin (e.g. also example), the app crashes at startup when no Bluetooth adapter exists in the system. 

I seems to be a null pointer at some point in initialisation code.

**Comment by fotiDim** [comment]
Thanks for the report. Fixed by https://github.com/Navideck/universal_ble/pull/20

**Comment by Regular89** [comment]
Hi, 
The log issue on my side was:
√ Built build\windows\x64\runner\Debug\universal_ble_example.exe. Error connecting to the service protocol: failed to connect to http://127.0.0.1:50910/....
So thanks @fotiDim for the fix, it worked for me. 
 
However because there is not a new version, I had to apply the patch by hand instead of "flutter pub upgrade".
Any idea when the new version will be released ? 
I saw that I can reference the git repo directly but I would rather avoid this.
